napari/util/misc.py

```diff
@@ -5,6 +5,12 @@
 import itertools
 
 
+def str_to_rgb(arg):
+    """Convert an rgb string 'rgb(x,y,z)' to a list of ints [x,y,z].
+    """
+    return [int(x) for x in arg.split('(')[1].split(')')[0].split(',')]
```

Comment:
would prefer to use a slightly more complex scheme which allows you to include whitespace between each comma

```suggestion
    return list(map(int,
                    re.match(r'rgb\((\d+),\s*(\d+),\s*(\d+)\)', arg).groups()))
```

also make sure to `import re` at the top

Reply:
also would prefer for this to be in some util file under `resources`

Reply:
discussed in person, will keep this one here for now, but made the regex change, nice suggestion!